src/test/java/com/google/api/codegen/testsrc/library_gapic.yaml

```diff
@@ -248,9 +248,13 @@ interfaces:
       - shelf.name=Math
       - series_uuid.series_string=xyz3141592654
       output_specs:
-      - print:
-        - "book names %s"
-        - $resp.book_names
+      - loop:
+          variable: book_name
+          collection: $resp.book_names
+          body:
+            print:
+              - "book name: %s"
```

Comment:
nit: Here I would have text that's slightly friendlier, like `"Title: %s"` or `"Book: %s"`